site-structure.json

```diff
@@ -5,6 +5,16 @@
         "sections":
         [
             {
+                "sectionName": "API",
+                "pages":
```

Comment:
There are two other API pages further down: ChangeApplier and DOM Binder. Should they be moved into this section?